ci/owncloud/upload_to_owncloud.sh

```diff
@@ -17,6 +17,7 @@ usage() {
     echo "      -h|--help - display this help."
     echo "      -v|--verbose - enable verbosity"
     echo "      -u|--url - webDAV URL (default https://ftp.essensium.com/owncloud/remote.php/dav/files/USERNAME)"
+    echo "      --direct - direct upload without first uploading to a temporary directory (default false)"
```

Comment:
I still don't really understand why you'd prefer to upload directly there, but ok.

Not ok though: the help message says the default is false, but you set it to true.

Reply:
It is mainly to avoid the dir/dir issue.. and it is confusing and unnecessary to upload to a temp location when uploading logs - if the upload fails at some point, the logs we did succeed to upload are still interesting, so direct upload IMO is the correct approach in this case.

Reply:
The default is empty, which means it is not true. I will add a default so it is clearer.